ui/raidboss/data/05-shb/raid/e5s.js

```diff
@@ -401,13 +403,16 @@
       },
       '~effectNames': {
         'System Shock': 'Électro-choc',
+        'Surge Protection': 'Parafoudre',
         'Hated of Levin': 'Malédiction du Patriarche fulgurant',
+        'Fury\'s Bolt': 'Boules de foudre',
         'Electrified': 'Électrocution irradiante',
         'Damage Down': 'Malus de dégâts',
       },
     },
     {
       'locale': 'ja',
+      'missingTranslations': true,
```

Comment:
Same question here.

Reply:
I understand that this line `'missingTranslations'`: true, should be there, when there are lines to translate. Am I mistaken?
![e5s ja](https://user-images.githubusercontent.com/61565834/84476775-ed9c5300-ac8e-11ea-931e-ba72aa5f7207.PNG)